app/src/main/res/values-w820dp/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pump_commerror_label">Connection Error</string>
```

Comment:
Why would this string be different on different device configurations? It seems to be the same for all?